dockerfiles/Dockerfile-zkapp-test-transaction

```
@@ -1,10 +1,10 @@
-ARG image=debian:stretch-slim
+ARG image=debian:bullseye-slim
 FROM ${image}
 
 # Run with `docker build --build-arg deb_version=<version>`
 ARG deb_version
-ARG deb_codename=stretch
-ARG deb_release=alpha
+ARG deb_codename=bullseye
+ARG deb_release=unstable
```

Not sure about the `deb_release` suggestion below ...
```suggestion
ARG deb_codename=buster
ARG deb_release=stable